pipelines/h4c/post_processing/v6/post_lstbin_frate/task_scripts/do_PSPEC.sh

```diff
@@ -16,7 +16,7 @@ spw_ranges="${5}"
 jd=$(get_jd $fn)
 int_jd=${jd:0:7}
 if [[ "$int_jd" == *"."* ]]; then
-  jd=`echo ${fn} | grep -o "[0-9]\{1,2\}.[0-9]\{5\}"`
+  jd=`echo ${fn} | grep -o "[0-9]\{1,2\}.[0-9]\{7\}"`
```

Comment:
Should this change actually be here? It looks like a validation-specific thing.

Reply:
Ah, yes, these were all on the fly changes we made to make the eor sim run.  I've reverted all these changes.